Create ui-c8y-1023-53-2-fixed-the-typeahead-for-the-software-type-selection-when-filtering-by.md (#4371)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-53-2-fixed-the-typeahead-for-the-software-type-selection-when-filtering-by.md

---
date: ""

> title: Fix software type filter losing selection in typeahead

Reviewer: ```suggestion
title: Software type filter now maintains selected value
```

> title: fixed typeahead for software type selection when filtering by type

Reviewer 2: ```suggestion
title: Fix software type filter losing selection in typeahead
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5654
version: 1023.53.2
---

> When filtering software by type, the typeahead field did not consistently preserve or display the selected software type. In some cases, the selection state became inconsistent, making filtering behavior confusing. This issue has been resolved - the software type typeahead now correctly maintains the selected value and reliably filters results based on the chosen software type.

Reviewer: ```suggestion
When filtering software by type, the typeahead field did not consistently preserve or display the selected software type. In some cases, the selection state became inconsistent, making filtering behavior confusing. This issue has been resolved. The typeahead field now correctly maintains the selected value and reliably filters results based on the chosen software type.
```

> When filtering software by type, the typeahead feature in the software type selection field was not functioning correctly, making it difficult to quickly find and select the desired software type. The typeahead functionality has now been fixed and will properly display matching software types as you type, allowing you to efficiently filter and select software types without having to manually scroll through the entire list. This improvement streamlines the software filtering workflow and reduces the time needed to locate specific software types in your installations.

Reviewer 2: ```suggestion
When filtering software by type, the typeahead field did not consistently preserve or display the selected software type. In some cases, the selection state became inconsistent, making filtering behavior confusing. This issue has been resolved - the software type typeahead now correctly maintains the selected value and reliably filters results based on the chosen software type.
```
